collectors/python.d.plugin/example_prometheus/example_prometheus.chart.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+# Description: example netdata python.d module
+# Author: Put your name here (your github login)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from copy import deepcopy
+from random import SystemRandom
+
+from bases.FrameworkServices.PrometheusService import PrometheusService, get_metric_by_name
+
+disabled_by_default = True
+
+
+priority = 90000
+
+ORDER = [
+    'jobs_processed',
+    'jobs_unprocessed',
+    'memory',
+]
+
+CHARTS = {
+    'jobs_processed': {
+        'options': [None, 'Jobs Processed Rate', 'jobs/s', 'jobs', 'example.jobs_processed', 'line'],
+        'lines': [
+            ['example_jobs_processed', 'processed', 'incremental'],
+        ]
+    },
+    'jobs_unprocessed': {
+        'options': [None, 'Jobs Unprocessed', 'jobs', 'jobs', 'example.jobs_unprocessed', 'line'],
+        'lines': [
+            ['example_jobs_unprocessed', 'unprocessed'],
+        ]
+    },
+    'memory': {
+        'options': [None, 'Memory Usage', 'KiB', 'memory', 'example.memory', 'line'],
+        'lines': [
+            ['example_memory_used_bytes', 'usage', 'absolute', 1, 1024],
+        ]
+    },
+}
+
+METRICS_SAMPLE = '''
```

Comment:
This is not valid OpenMetrics data. This is valid Prometheus text format data though. I'd suggest removing mentions of OpenMetrics here to avoid confusion.

You can use the same general approach for OpenMetrics, the only change needed is using the openmetrics parser.